docs/reference/src/language/builtins/globals.md

```diff
@@ -19,6 +19,8 @@ the selected style e.g. fluent, cupertino, material, or qt.
 -   **`selection-background`** (_out_ _brush_): Defines the background brush that is used to highlight a selection such as a text selection.
 -   **`selection-foreground`** (_out_ _brush_):  Defines the foreground brush that is used for content that is displayed on `selection-background` brush.
 -   **`border`** (_out_ _brush_): Defines the brush that is used for borders such as separators and widget borders.
+-   **`dark-color-scheme`** (_in_ _out_ _bool_): Read this property to determine if the currently selected palette follows a dark color scheme.
+    Set this property to force a dark or light color scheme. All styles except for the Qt style support setting a dark or light color scheme.
```

Comment:
do the -dark style support setting a light color style and vice versa?

Reply:
Fortunately yes. Basically they work like this:

```
global FluentPalette {
    in-out property <bool> dark-color-scheme: ColorSchemeSelector.dark-color-scheme;
```

and `ColorSchemeSelector`'s property is either true or false depending on the chosen style (-light/-dark suffix) or it defaults to `SlintInternal.dark-color-scheme` (the builtin that calls the window adapter). But everything in the fluent style uses the above property, so the exposed `Palette.dark-color-scheme`'s two-way binding ends up breaking up the binding to `ColorSchemeSelector` and setting the forced value (after which it doesn't react to system changes anymore).